lib/src/pages/home.dart

```diff
@@ -5,6 +5,51 @@ import "package:rover_dashboard/models.dart";
 import "package:rover_dashboard/pages.dart";
 import "package:rover_dashboard/widgets.dart";
 
+/// A widget to view timer 
+/// Can also stop and start timer
+class Timer extends StatelessWidget {
+	@override
+	Widget build(BuildContext context) => ProviderConsumer<HomeModel>.value(
+		value: models.home,
+		builder: (model) => (model.timer == null || model.timer!.timeLeft < Duration.zero)
+      ? Container()
+      : Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            Text(model.timer!.name,
+              style: const TextStyle(color: Color.fromRGBO(255, 255, 255, 1), fontSize: 18),
+            ),
+            SizedBox(width: model.timer!.underMin ? 8 : 0),
+            Container(
+              padding: const EdgeInsets.all(8),
+              color: model.timer!.underMin ? const Color.fromRGBO(0, 0, 0, 1) : const Color.fromRGBO(0, 0, 0, 0),
+              child: Text("${model.timer?.timeLeftFormatted}",
+                style: model.timer!.underMin ? const TextStyle( 
+                  color:  Color.fromRGBO(179, 0, 0, 1), 
+                  fontSize: 24,
+                  fontWeight: FontWeight.bold,
+                ) 
+                : const TextStyle( 
+                  color: Color.fromRGBO(250, 255, 255, 1), 
+                  fontSize: 20,
+                ),
+              ),
+            ),
+            SizedBox(width: model.timer!.underMin ? 8 : 0),
+            ElevatedButton(
+              onPressed: () => model.timer!.paused ? model.resumeTimer() : model.pauseTimer(),
+              child: model.timer!.paused ? const Text("Resume") : const Text("Pause"), 
+            ),
+            const SizedBox(width: 8),
+            ElevatedButton(
+              onPressed: () => model.stopTimer(),
+              child: const Text("Delete"), 
+            ),
+          ],
+        ),
+    );
```

Comment:
```suggestion
      : AnimatedScale(
        scale: model.timer!.underMin ? 1.1 : 1, 
        duration: const Duration(milliseconds: 500), 
        child: Row(
          mainAxisAlignment: MainAxisAlignment.center,
          children: [
            Text("${model.timer!.name}: ",
              style: context.textTheme.headlineSmall!.copyWith(color: context.colorScheme.onPrimary),
            ),
            const SizedBox(width: 4),
            Text("${model.timer?.timeLeftFormatted}",
              style: model.timer!.underMin
                ? context.textTheme.headlineSmall!.copyWith(
                  color: context.colorScheme.error,
                  fontWeight: FontWeight.bold,
                )
                : context.textTheme.headlineSmall!.copyWith(color: context.colorScheme.onPrimary),
            ),
            const SizedBox(width: 8),
            ElevatedButton(
              onPressed: model.timer!.paused ? model.resumeTimer : model.pauseTimer,
              child: model.timer!.paused ? const Text("Resume") : const Text("Pause"), 
            ),
            const SizedBox(width: 8),
            ElevatedButton(
              onPressed: model.stopTimer,
              child: const Text("Cancel"), 
            ),
          ],
        ),
      ),
  );
```

Some very small changes here: 
1. Instead of making your own colors, use the ones from the [`ColorScheme`](https://api.flutter.dev/flutter/material/ColorScheme-class.html). The Binghamton green color is `primary`, so the white text would be `onPrimary`. The real way to get the color scheme is `Theme.of(context).colorScheme`, but I made an extension in `lib/widgets.dart` that lets you do `context.colorScheme`
2. Removed the black box, which lets us have sized boxes with constant widths
3. Similar to (1), instead of making our own `TextStyle`s, just take the ones from [`TextTheme`](https://api.flutter.dev/flutter/material/TextTheme-class.html), which can be accessed by `Theme.of(context).textTheme`, or in our case, just `context.textTheme`. Usually they should have the right color, I'm not sure why I had to override it back to white, but you see you can use `.copyWith` to do so.
4. Instead of making a lambda function `() => model.stopTImer()`, you can just refer to the function itself, `model.stopTimer`. Not using the parentheses means you're not calling the function, you're pointing to it.